### ISSUE TITLE
In Bumper mode the hit-rates numbers intersect with the dividers

### ISSUE THREAD

**Issue by QuackQuackBlah** [PoochyEXE/PoochyEXE.github.io#36]
<img width="359" alt="Screen Shot 2021-09-11 at 9 44 42 AM" src="https://user-images.githubusercontent.com/77398316/132955112-49ec78e4-8856-42d2-a07e-d157277b5d04.png">

By the way, I love what you're doing with the game! This is an insanely great improvement for micro-optimizing the game, namely where the drop should happen! (also the rubber band balls are quite neat too!)

**Comment by PoochyEXE** [comment]
Yep, I know about this. The opaque white outlines around the numbers are currently my workaround for it, so that the numbers at least don't blur into the pegs.
The problem is most targets have pegs to the left and right of them, and some have pegs above, below, or both, so there's no good one-size-fits-all place to put the numbers relative to the targets.
I'm glad to hear you're enjoying the new version though, thanks!
